pkg/scope/provider.go

```diff
@@ -152,6 +153,10 @@ func NewProviderClient(cloud clientconfig.Cloud, caCert []byte, logger logr.Logg
 		return nil, nil, "", fmt.Errorf("create providerClient err: %v", err)
 	}
 
+	ua := gophercloud.UserAgent{}
+	ua.Prepend(fmt.Sprintf("cluster-api-provider-openstack/%s", version.Get().String()))
```

Comment:
Is there benefit in using the default user agent and simply prepending to it? That way we'll also get the gophercloud version in use. Perhaps that's overkill...?

Reply:
Personally I mimicked the patter that was used by the CPO so that's how I got where I got.